build.gradle

```diff
@@ -1,15 +1,15 @@
 plugins {
     id 'java'
     id 'io.freefair.lombok' version '6.6.1'
-    id 'org.springframework.boot' version '2.6.2'
+    id 'org.springframework.boot' version '2.6.14'
     id 'io.spring.dependency-management' version '1.1.0'
     id 'jacoco'
     id 'org.sonarqube' version '3.3'
     id 'maven-publish'
 }
 
 ext {
-    springCloudVersion = '2021.0.0'
+    springCloudVersion = '2021.0.5'
```

Comment:
Do we still use this dependency ?

Reply:
I'm talking about `spring-cloud-dependencies` and `spring-cloud-starter-openfeign`

Reply:
Good catch. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/149/commits/734e525f85cf3502ff1bb4c991a3bf58d3083ef9.

Reply:
Thanks